package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "vscode-commons",
   "displayName": "Red Hat common extension",
-  "description": "",
+  "description": "Telemetry Middleware",
   "version": "0.0.1",
   "publisher": "redhat",
-  "segmentWriteKey": "MXM7iv13sVaCGqOhnQEGLZxhfy6hecYh",
+  "segmentWriteKey": "Ga2frlNQ8nsYjF8PEGsbdjhCy3VJdncB",
   "engines": {
-    "vscode": "^1.37.0"
+    "vscode": "^1.51.0"
```

Comment:
might be too high of a requirement for Che/Theia. Something to check.

Reply:
Ya, we may need to check on this. 
Can you guide me on how we can verify this and are there anymore IDE/extensions for which we should verify the compatibility

Reply:
@benoitf what's the minimal supported vscode version in Cheia?

Reply:
so 1.50 it is

Reply:
ack

Reply:
done